crypto/rand/rand_lcl.h

```diff
@@ -107,6 +107,8 @@ typedef struct rand_drbg_ctr_st {
     unsigned char KX[48];
 } RAND_DRBG_CTR;
 
+/* forward declaration */
+typedef struct rand_pool_st RAND_POOL;
```

Comment:
This seems unnecessary.  Just make sure to include `"internal/rand_int.h"` before including `"rand_lcl.h"`

My reasoning is that code copying, even one line, is bad and should be avoided if possible (it's not always possible, but I think that at least an effort should be made), because a few years down the line, fixes have been made in one spot but not the other, and no one notices for a long while...  This has happened before in numerous projects and will most certainly happen again, so let's avoid it when we can, yeah?

Reply:
Works with me :-)